configs/components/_base-ruby-selinux.rb

```diff
@@ -21,6 +21,12 @@
   pkg.version '3.3'
   pkg.sha256sum 'acfdee27633d2496508c28727c3d41d3748076f66d42fccde2e6b9f3463a7057'
   pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
+elsif platform.name.start_with?('debian-12')
```

Comment:
What made us add these conditions, in the past we have added platforms but never required a change here. Just want to enhance my knowledge.

Reply:
@amitkarsale building against a much older libselinux will result in broken runtime bindings. [See PA-5632](https://perforce.atlassian.net/issues/PA-5632) for details on why the 3.3 was added. While we don't actively test selinux on Ubuntu or Debian (I would like to see that feature added in the future) this ensures the build works properly and built against the vendor shipped version.

As to why the vendor libselinux devel package isn't used instead of downloading the source, I'm not sure. It's mentioned in PA-5632 to keep using source.

Reply:
@amitkarsale Can you link to the failed pipeline please? I don't see any failures.